webapp/src/viewModel/cardTree.ts

```diff
@@ -12,19 +12,19 @@ import {OctoUtils} from '../octoUtils'
 interface CardTree {
     readonly card: Card
     readonly comments: readonly CommentBlock[]
-    readonly contents: readonly IContentBlock[]
+    readonly contents: readonly (IContentBlock|IContentBlock[])[]
```

Comment:
same type problem here, it is hard to understand, maybe we can express this in other way.

Reply:
Done